scripts/onboarding/set-variables.CanadaPubSecALZ.bat

```diff
@@ -0,0 +1,54 @@
+@echo off
+REM // ----------------------------------------------------------------------------------
+REM // Copyright (c) Microsoft Corporation.
+REM // Licensed under the MIT license.
+REM //
+REM // THIS CODE AND INFORMATION ARE PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
+REM // EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED WARRANTIES 
+REM // OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+REM // ----------------------------------------------------------------------------------
+
+REM Azure AD tenant GUID
+set DEVOPS_TENANT_ID=df968f64-5f44-4e42-937b-47fb792ee373
```

Comment:
Not sure here also if we need to change any of this data?

Reply:
we don't use devops anymore so I'm not sure this is relevant. The 'on board' document only describes how to use devops to deploy.